openapi-v1.yaml

```diff
@@ -6385,87 +6785,222 @@ paths:
           description: pagination limit for assets
           type: integer
           required: false
+        - name: namespace
+          in: query
+          description: namespace to search for
+          required: false
+          type: string
+        - name: search
+          in: query
+          description: search string that will look at name, namespace or description fields
+          required: false
+          type: string
+        - name: orderby
+          in: query
+          description: sort by which field valid values include last_accessed, size, name
+          required: false
+          type: string
+        - name: tag
+          in: query
+          description: tag to search for, more than one can be included
+          required: false
+          type: array
+          collectionFormat: multi
+          items:
+            type: string
+        - name: exclude_tag
+          in: query
+          description: tags to exclude matching array in results, more than one can be included
+          required: false
+          type: array
+          collectionFormat: multi
+          items:
+            type: string
+        - name: file_type
```

Comment:
Since we introduced GroupMemberType, i'm wondering if we should use GroupMemberType here instead of file_type and exclude_file_type (which is an array-only property).
This way we can filter the results by any type (including group type, which is missing from the filters at the moment).

Reply:
Yes this should be member type.